workers/unity/Packages/io.improbable.gdk.testutils/MockWorld.cs

```diff
@@ -22,18 +19,18 @@ public struct Options
 
         public WorkerInWorld Worker { get; private set; }
         public MockConnectionHandler Connection { get; private set; }
-
         public EntityGameObjectLinker Linker { get; private set; }
 
         private readonly HashSet<GameObject> gameObjects = new HashSet<GameObject>();
+        public MockCommandSender CommandSender { get; private set; }
```

Comment:
Does this need to be public? Is there a way to only interact with the `CommandSystem`?

Reply:
I guess the point is why should we have to GenerateResponses in the test, instead of it being done for us in between steps?

Reply:
My thinking was this allows you to simulate the responses arriving at non consecutive frames or even not arriving at all. At the same time I feel like this makes it _too_ different to the way you set up other things like component updates, events, etc.
Since everything else is done through `world.Connection.Addxxx()` whereas commands need to be called through `world.CommandSender.SendCommand(xxx)` or `world.GetSystem<CommandSystem>().SendCommand(xxx)`

Reply:
> allows you to simulate the responses arriving at non consecutive frames or even not arriving at all

Ah yes. That makes sense 👍